config/swagger.yaml

```diff
@@ -123,15 +147,23 @@ definitions:
       version:
         type: integer
         format: int64
-  Header:
+      pow:
+        type: string
+      txs:
+        type: string
```

Comment:
This type looks wrong. Reference https://github.com/aeternity/epoch/wiki/Block
I expect list of transactions.